packages/devtools/test/settings_controller_test.dart

```diff
@@ -20,19 +22,40 @@ void main() async {
     test('entering', () async {
       await env.setupEnvironment();
 
-      String sdkVersion;
+      FlutterVersion flutterVersion;
       List<Flag> flags;
-      settingsController =
-          SettingsController(onSdkVersionChange: (isAnyFlutterApp) {
-        sdkVersion = isAnyFlutterApp;
-      }, onFlagListChange: (flagList) {
-        flags = flagList.flags;
-      });
-      expect(sdkVersion, null);
+      settingsController = SettingsController(
+        onFlutterVersionChanged: (version) {
+          flutterVersion = version;
+        },
+        onFlagListChanged: (flagList) {
+          flags = flagList.flags;
+        },
+      );
+
       expect(flags, null);
+      expect(flutterVersion, null);
 
       await settingsController.entering();
-      expect(sdkVersion, 'Dart SDK Version: ${serviceManager.sdkVersion}');
+
+      try {
+        final flutterVersionResponse = await serviceManager.getFlutterVersion();
+        final expectedFlutterVersion =
+            FlutterVersion.parse(flutterVersionResponse.json);
+        expect(
+          _flutterVersionEquals(flutterVersion, expectedFlutterVersion),
+          isTrue,
+        );
+      } catch (e) {
+        expect(flutterVersion, isNull);
+        expect(
+          e.toString(),
+          equals('Exception: There are no registered methods for service'
```

Comment:
Add a TODO to remove this catch block once Flutter stable supports the service. Provide a date on the TODO when that is expected to happen.